package/libsolv.spec.in

```diff
@@ -190,83 +165,80 @@ Group:          Development/Languages/Python
 Python3 bindings for libsolv.
 
 %package -n perl-solv
-Requires:       perl = %{perl_version}
 Summary:        Perl bindings for the libsolv library
 Group:          Development/Languages/Perl
+Requires:       perl = %{perl_version}
 
 %description -n perl-solv
 Perl bindings for libsolv.
 
 %prep
-%setup -n libsolv-%{version}
+%setup -q
```

Comment:
Thank you! 💯

Reply:
Now time to change to %autosetup :P

Reply:
They didn't backport new macro stuff to SLE 11 like we did for RHEL 5, so they still can't do that.